client/my-sites/domains/components/domain-warnings/pending-gapps-tos-notice-action.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Fragment, useState } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PendingGappsTosNoticeDialog from './pending-gapps-tos-notice-dialog';
+
+function PendingGappsTosNoticeAction( props ) {
+	const [ dialogVisible, setDialogVisible ] = useState( false );
+	const onFixClickHandler = e => {
+		e.preventDefault;
```

Comment:
We're missing parentheses here.

Reply:
It is not required: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Basic_syntax

I have added it as I thought Prettier would have taken care of it. Not sure what the standard is.

Reply:
I think we're not talking of the same thing 😉.

I was referring to `e.preventDefault;`, which is invalid.

Reply:
Haha, fixing.